root/static/scripts/edit/MB/Control/Bubble.js

```diff
@@ -332,4 +332,4 @@ MB.Control.initializeBubble = function (bubble, control, vm, canBeShown) {
     return bubbleDoc;
 };
 
-module.exports = MB.Control;
+export default MB.Control;
```

Comment:
This breaks an import in root/static/scripts/place.js. I suggest `export const initializeBubble = MB.Control.initializeBubble;` and no default export.